src/Dropdown.directive.js

```diff
@@ -67,7 +67,7 @@ const temporaryHideAllDropdowns = (
   dropdownId,
 ) => {
   const eventParentIds = event?.path?.map(({ id }) => id) || [];
-  const isEventFromInsideDropdown = eventParentIds.includes(dropdownId);
+  const isEventFromInsideDropdown = eventParentIds.includes(dropdownId) || !!event.target.closest(".dropdown");
```

Comment:
I guess this will allow multiple drop downs to be open at once ? But either way it's fine

Reply:
Don't think so. 
is just that the eventParentIds seems to not be working fine in some scenario, not sure why.  the modification just check the the src of the event is inside dropdown using the attribute `dropdown-id`that is on all dropdowns.